### DISCUSSION TITLE
Skr mini e3 v3.0 with error message media init fail

### DISCUSSION THREAD

**Discussion by Bubbleslm12** [community/community#55974] [New to GitHub]
Hi is anyone having the issues I'm having with a skr mini e3 v3.0 

I want to use a cable Extender and converter to make it from a micro sd card to a full size sd card for my ender 3

Before the firmware update I was get sd init fail 

But since the firmware update I'm no getting media init fail
Is anyone else having this issue 

Thanks

**Comment by queenofcorgis** [comment]
Hi @Bubbleslm12 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this topic. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
